Americastestkitchen/mise-ui
# chore: delete old sticker files and consolidate type usage

### What does this PR do?
Deletes the old .js Sticker files and correctly points imports to .tsx files. Also consolidates Sticker type usage to pull from `Carts.d.ts`.

### How should I review this PR?
1. Verify that imports correctly point to .tsx files and type passes.
2. Pay attention to the updates in `Carts.d.ts` and verify the type values of `icon` and `type` are correct.
3. Check the HeroCard test, I updated the expectation to be 2 instead of 3 because I couldn't find how the 3rd sticker was rendered.
